db/assimilator.go

```diff
@@ -21,29 +19,7 @@ func (c *DatabaseContext) watchDocChanges() {
 				if c.Shadower != nil {
 					c.Shadower.PushRevision(doc)
 				}
-			} else {
-				if c.autoImport {
```

Comment:
Can we remove c.autoImport altogether?

Reply:
Merged ImportDocs and autoImport into the same variable throughout the codebase.  The legacy config support is provided via keeping the name in the json as `import_docs`